utils/src/main/kotlin/Utils.kt

```diff
@@ -192,7 +192,7 @@ fun joinNonBlank(vararg strings: String, separator: String = " - ") =
  * Normalize a string representing a [VCS URL][vcsUrl] to a common string form.
  */
 fun normalizeVcsUrl(vcsUrl: String): String {
-    var url = vcsUrl.trimEnd('/')
+    var url = vcsUrl.trim().trimEnd('/')
```

Comment:
Commit message: s/normalizeVcsUr()/normalizeVcsUrl()/

Reply:
Regarding the actual change, I'm not generally against doing that trimming, but I wonder whether we should do it somewhere else / in the caller. Because strictly speaking this function should normalize valid URLs, but something with a leading space is not a valid URL. So, where exactly did the URL with the leading space come from?

Reply:
From a curation =:-)

Reply:
I've updated the commit message.

Reply:
> Do you prefer to have it there?

No, let's the keep it here.